src/validation/drydep/radius_for_moment.cpp

```diff
@@ -0,0 +1,52 @@
+// mam4xx: Copyright (c) 2022,
+// Battelle Memorial Institute and
+// National Technology & Engineering Solutions of Sandia, LLC (NTESS)
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <iostream>
+#include <mam4xx/drydep.hpp>
+#include <skywalker.hpp>
+#include <validation.hpp>
+
+using namespace skywalker;
+using namespace mam4;
+
+void radius_for_moment(Ensemble *ensemble) {
+
+  // Run the ensemble.
+  ensemble->process([=](const Input &input, Output &output) {
+    if (!input.has("moment")) {
```

Comment:
Not at all important, but repetitive code like this is ripe for a CPP macro, e.g.

```
#define CHECK_FOR_INPUT(name) \
if (!input.has(#name)) { \
  std::cerr << "Required name: " << #name << std::endl; \
  exit(1); \
}

...
  CHECK_FOR_INPUT("moment")
  CHECK_FOR_INPUT("sig_part")
  CHECK_FOR_INPUT("radius_part")
  CHECK_FOR_INPUT("radius_max")
```

But this is largely a matter of taste.

Reply:
Totally, we should have done that a long time ago.  I think we need to go back and remove this repetitive code. In fact, I vaguely remember now that we have a macro defined somewhere that already does just this.

Reply:
If we don't already have one, you could stick such a macro in `src/validation/validation.hpp`. And you might change the header guard at the top of the file to `MAM4XX_VALIDATION` instead of `HAERO_VALIDATION`. :-)

Reply:
Remove as discussed.